salt/metalk8s/roles/bootstrap/init.sls

```diff
@@ -1,2 +1,14 @@
-Deploy bootstrap:
-  test.succeed_without_changes
+include:
+  - metalk8s.node.grains
+  - metalk8s.kubernetes.kubelet.standalone
+  - metalk8s._internal.preflight
+  - metalk8s.registry.populated
+  - metalk8s.repo.installed
+  - metalk8s.salt.master
+  - metalk8s.roles.minion
+  - metalk8s.salt.master.accept_keys
+{%- if salt['config.get']('file_client') == 'remote' %}
```

Comment:
Hmh, not sure that's the best approach.

How about a `bootstrap/no-master.sls`, and have `bootstrap/init.sls` `include` `.no-master`, and use the appropriate SLS (in bootstrap script or not) where applicable.

Reply:
Yes we can I basically did it when we haven't yet the directory in `roles`